pkgs/tools/misc/mprocs/default.nix

```
@@ -0,0 +1,22 @@
+{ lib, fetchFromGitHub, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "mprocs";
+  version = "0.2.2";
+
+  src = fetchFromGitHub {
+    owner = "pvolok";
+    repo = pname;
+    rev = "refs/tags/v${version}";
```

I don't think `refs/tags` is needed
```suggestion
    rev = "v${version}";
```